### PR TITLE
Templates for multiple IPs per NIC feature

### PR DESCRIPTION
**Pull request by anavinahar**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use resourceGroup().location for resource locations
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/bp-checklist.md

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

*
*
*

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@anavinahar__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (annahar). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by anavinahar** [comment]
@singhkays  Hey Kay, I can't fix this error as it only occurs when a subscription has not been enabled for this feature. One the test subscription is enabled, it should work. I enabled it for "f42b4319-067a-4548-a650-33a1553b3a42", can you rerun the rest?

File: 101-vm-multiple-ip/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Create a VM with multiple IPs
```
Comment (singhkays) [inline 101-vm-multiple-ip/README.md]: Can you call this folder  101-vm-multiple-ipconfig ?

File: 101-vm-multiple-ip/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-vm-simple-linux%2Fazuredeploy.json" target="_blank">
```
Comment (singhkays) [inline 101-vm-multiple-ip/README.md]: Links need to be updated to this template

File: 101-vm-multiple-ip/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,258 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "storageAccountName": {
```
Comment (singhkays) [inline 101-vm-multiple-ip/azuredeploy.json]: See best practices, don't ask the user for storage account name

File: 101-vm-multiple-ip/azuredeploy.json [hunk 4]
```diff
@@ -0,0 +1,258 @@
...
+             "allowedValues": [
+               "2008-R2-SP1",
+               "2012-Datacenter",
+               "2012-R2-Datacenter",
+               "2016-Nano-Server",
+               "2016-Datacenter-with-Containers",
+               "2016-Datacenter",
+               "12.04.5-LTS",
+               "14.04.5-LTS",
+               "15.10",
```
Comment (singhkays) [inline 101-vm-multiple-ip/azuredeploy.json]: Only use 16.04 LTS ubunutu, other images get removed from Azure

File: 101-vm-multiple-ip/azuredeploy.json [hunk 5]
```diff
@@ -0,0 +1,258 @@
...
+          "dnsLabelPrefix1": {
+            "type": "string",
+            "metadata": {
+              "description": "DNS for PublicIPAddressName2"
+        }
+    },
+        "OSVersion": {
+             "type": "string",
+             "allowedValues": [
+               "2008-R2-SP1",
```
Comment (singhkays) [inline 101-vm-multiple-ip/azuredeploy.json]: skip WS 2008

**Comment by singhkays** [comment]
lgtm, merged

**Pull request merged**